config/locales/en.yml

```diff
@@ -123,10 +123,12 @@ en:
       linestring: "Line editor"
       polygon: "Area editor"
       clear_map: "Clear map"
+      remove_point: "Remove point"
       copied_location_to_clipboard: "Copied location to clipboard"
     modal:
       load: "Load"
       cancel: "Cancel"
     messages:
       baselayer_missing: "There is no baselayer available!"
       zoom_in_more: "Zoom in to view objects."
+      modify_start: "Hold down <code>ALT</code> and click on a vertex to delete it."
```

Comment:
The message can be improved for the following patterns.
* PC:
  * Linux/Windows: `ALT`
  * macOS: `option`
* Mobile (iOS/Android):
  * `click` => `tap`

Reply:
Do you know how to identify "mobile"? Because in this case the notification doesn't need to show up.

Regarding the key names, that's right, but this can be adjusted during translation anyway.

Reply:
I also tested once to set the "edit" button to "disabled" ... I felt it looks a bit ugly, that's why I didn't disable it ;-)

Reply:
> Do you know how to identify "mobile"? Because in this case the notification doesn't need to show up.

I don't know about it...
From glance of openlayers code, `'ontouchstart' in window` seems to be used.
https://github.com/search?q=repo%3Aopenlayers%2Fopenlayers+ontouchstart&type=code

---

> Regarding the key names, that's right, but this can be adjusted during translation anyway.

Okay.

---

> I also tested once to set the "edit" button to "disabled" ... I felt it looks a bit ugly, that's why I didn't disable it ;-)

Okay, I see.
Then skipping it is fine.

Reply:
This is how the disabled button would look like:

![image](https://github.com/gtt-project/redmine_gtt/assets/227762/6d2e5e7a-81d9-46c5-8a2c-0551dbfeb071)

Reply:
[Screencast from 2024-06-05 12-41-41.webm](https://github.com/gtt-project/redmine_gtt/assets/227762/d1f484da-84e2-47bc-8762-4c2c4d58153e)

How about hiding the buttons when they make no sense?

Reply:
@sanak , I also added support to handle messages better:
- it checks for mobile devices
- it checks for Mac (couldn't test this)
- it does not show a message when the geometry is of type `Point`, because then the delete vertex does not make sense.

Reply:
@dkastl Okay, great!
I confirmed that mobile device (now iPhone) and Mac OS, so I approve this.
Thanks a lot!

Reply:
There is one case, that the edit and delete buttons don't hide again, when a user stats drawing a line or polygon and before finising the feature clicks on another button. In this case no feature exists, but the edit and delete control won't be hidden.

It's tricky to also cover this case, and I think it's acceptable, so I will merge as it is.